### PR TITLE
Release 1.8.0

### PR DESCRIPTION
**Pull request by leoner**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: History.md [hunk 1]
```diff
@@ -1,5 +1,38 @@
...
+## 2017-09-11, Version 1.8.0, @leoner
+
+### Notable changes
+
+* **feature**
+  * support app.httpclient and agent.httpclient auto set tracer
+* **fix**
+  * should extends from egg-core BaseContextClass
+* **document**
+  * finish logger.md translation in English @gztchan
```
Comment (fengmk2) [inline History.md]: `@gztchan` 不需要写

Comment (leoner) [reply History.md]: 额, 好的, 我看下面关于这部分, 都写了这个.

**Comment by fengmk2** [comment]
我手动操作。

**Comment by fengmk2** [comment]
Released.

**Pull request merged**